GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/JavaMap.java

```diff
@@ -0,0 +1,33 @@
+package hackkerank.day3;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class JavaMap {
+    public static void main(String []argh)
+    {
+        Scanner in = new Scanner(System.in);
+        int numberOfEntries=in.nextInt();
+        in.nextLine();
+        Map<String, Integer> map = new HashMap<>(numberOfEntries);
+
+        for(int i=0;i<numberOfEntries;i++)
```

Comment:
I see missing whitespaces in almost all your code.... please refer to:
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace